src/encoded/locust/main.py

```diff
@@ -0,0 +1,40 @@
+import os
+import argparse
+import subprocess
+
+def main():
+    """
+    Entry point for locust load testing. Run with -h to see arguments
+    """
+    parser = argparse.ArgumentParser(
+        description='Locust Load Testing',
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument('config', help='path to config.json')
+    parser.add_argument('key', help='path to <env>.json')
+    parser.add_argument('--time', help='time to run test for, default 1m. Format: 10s, 5m, 1h, 1h30m etc.')
+    parser.add_argument('--clients', help='number of clients, default 10')
+    parser.add_argument('--rate', help='number of clients to hatch per second, default 10')
+    parser.add_argument('--lower', help='lower bound on time to wait between requests, default 1')
+    parser.add_argument('--upper', help='upper bound on time to wait between requests, default 2')
+  
+    args = parser.parse_args()
+
+    # set env variables (this is the only way to do this) and configure args
+    env_copy = os.environ.copy()
+    env_copy['LOCUST_CONFIG'] = args.config
+    env_copy['LOCUST_KEY'] = args.key
+    env_copy['LOCUST_ENV'] = args.key.split('.')[0]  # data.json -> data
```

Comment:
Should this be `os.path.splitext(os.path.basename(args.key))[0]`? I find it annoying there's not a better way with `os.path` to get at the part before the extinction, but so it goes. The thing you're doing, but the main difference is that by using the file operations you make yourself immune to someone using a file specifier, and you alert the human reader of your code to the fact that the `args.key` is a pathname, which I _think_ it is. :)